3.analysis/scripts/nbconverted/1.heatmaps.py

```diff
@@ -60,7 +60,7 @@
 # subsetting the genes
 
 # create dataframe containing the genes that passed an initial QC (see Pan et al. 2022) and their corresponding gene label and extract the gene labels
-gene_dict_df = pd.read_csv("../0.data-download/data/CRISPR_gene_dictionary.tsv", delimiter='\t')
+gene_dict_df = pd.read_parquet("../0.data-download/data/CRISPR_gene_dictionary.tsv", delimiter='\t')
```

Comment:
Will this cause an error? This is trying to open a `.tsv` file not a Parquet file